latex/ledger-spec.tex

```diff
@@ -360,16 +366,44 @@ \section{UTxO}
         \end{cases}\\
       &
       \begin{array}{lr@{~=~}l}
-        \where &\fun{releasing}~\var{c} & \DeregKey{c} \lor \RetirePool{c}\\
+        \where &\fun{releasing}~\var{c}
+          & c \in \DCertDeRegKey \cup \DCertRetirePool\\
         & d_{\min},~\lambda & \fun{decay}~pc\\
         &\delta & \slotminus{slot}{(allocs~(\cauthor c))}\\
       \end{array}\\
       \nextdef
-      & \fun{refunds} \in \PrtclConsts \to \Allocs \to \Allocs \to \Tx \to \Coin
-      & \text{total refunds for transaction} \\
-      & \refunds{pc}{stkeys}{stpools}{tx} =\\
+      & \fun{keyRefunds} \in \PrtclConsts \to \Allocs \to \Tx \to \Coin
+      & \text{key refunds for transaction} \\
+      & \keyRefunds{pc}{stkeys}{tx} =\\
       &   \sum\limits_{c \in \fun{dderegister}~tx} \refund{pc}{stkeys}{(\ttl{tx})}{c}\\
-      &   ~~~+ \sum\limits_{c \in \fun{dretire}~tx} \refund{pc}{stpools}{(\retire{c})}{c}
+      \nextdef
+      & \fun{lastEpoch} \in \Slot \to \Epoch
+      & \text{(abs.) slot of last epoch} \\
+      & \fun{lastEpoch}~{s} =  s - (s~\mathsf{div}~\slotsPer)
+      \nextdef
+      & \fun{decayed} \in
+      \PrtclConsts \to \Allocs \to \Slot \to \DCert \to \Coin
+      & \text{amount decayed since epoch} \\
+      & \decayed{allocs}{pc}{cslot}{c} =\\
+      & \begin{cases}
+        0 & \text{if not}~(\fun{releasing}~c)\\
+            0 & \text{if}~\cauthor c \notin allocs\\
```

Comment:
`allocs` is a map, `c` should not be in its domain (analogously in `refund`)

Reply:
thanks!